lottie/src/main/java/com/airbnb/lottie/parser/moshi/LinkedHashTreeMap.java

```diff
@@ -349,13 +349,11 @@ private void rebalance(Node<K, V> unbalanced, boolean insert) {
         int rightLeftHeight = rightLeft != null ? rightLeft.height : 0;
 
         int rightDelta = rightLeftHeight - rightRightHeight;
-        if (rightDelta == -1 || (rightDelta == 0 && !insert)) {
-          rotateLeft(node); // AVL right right
-        } else {
+        if (rightDelta != -1 && (rightDelta != 0 || insert)) {
```

Comment:
All of the moshi files need to be reverted

Reply:
All moshi changes reverted.